message DseChatMsg
{
	optional string 		uid		= 1;//发起者
	optional int32			head 	= 2;//头像
	optional string			name	= 3;//名字
	optional int32			level	= 4;//等级
	optional int32			viplevel = 5;//vip等级
	optional string			content	= 6;//聊天内容
	optional int32			broadcastid = 7;//广播ID
	repeated string			params		= 8;//参数列表
	optional string			chattime	= 9;//聊天时间
	optional int32			chattype	= 10;//1系统2世界聊天
}
